services/subnet_manager/src/test/java/com/futurewei/alcor/subnet/SubnetControllerTest.java

```diff
@@ -133,8 +133,8 @@ public void subnetCreateTest2 () throws Exception {
     public void subnetUpdateTest1 () throws Exception {
```

Comment:
Can we have a better name for the UTs, a recommended name should mention what the main test scenario is.

Reply:
Thanks for suggestions ~ now I make the method name as [Method name] _ [Test condition] _ [Expected return] form.